### PR TITLE
Nullable: Internal.Console, System.ValueType

### PR DESCRIPTION
**Pull request by buyaa-n**
Very easy PR

### CODE REVIEW

**Comment by safern** [comment]
Same as System.Object.ToString, if we decide it can return null, I think System.ValueType.ToString, do the same thing

@stephentoub ?

**Comment by stephentoub** [comment]
@buyaa-n, this PR has a bunch of unrelated commits in it.  Can you please rebase?

**Comment by stephentoub** [comment]
> if we decide it can return null, I think System.ValueType.ToString, do the same thing

Yes, they should match IMO.

**Comment by buyaa-n** [comment]
Sure rebased, seems final decision for ToString() not made, will wait for that

**Comment by safern** [comment]
@buyaa-n let's make ToString return nullable for now until a final decision is reached to not block ourselves on this, we should make progress: https://github.com/dotnet/coreclr/pull/23510#issuecomment-477734359

File: src/System.Private.CoreLib/src/Internal/Console.cs [hunk 1]
```diff
@@ -27,7 +28,7 @@ public static unsafe void Write(string s)
             }
         }
 
-        public static void WriteLine(string s) =>
+        public static void WriteLine(string? s) =>
```
Comment (stephentoub) [inline src/System.Private.CoreLib/src/Internal/Console.cs]: It's a little strange that `WriteLine(string)` accepts `null` but `Write(string)` does not.  That said, even though this type is "public", it isn't really, so it's not a big deal.

**Pull request merged**